sql/load_demo_data.sql

```diff
@@ -6099,8 +6295,12 @@ CREATE TABLE `oh_session_audit` (
   `SEA_LAST_MODIFIED_DATE` datetime DEFAULT NULL,
   `SEA_ACTIVE` tinyint(1) NOT NULL DEFAULT 1,
   PRIMARY KEY (`SEA_ID`),
-  KEY `SEA_US_ID_A` (`SEA_US_ID_A`),
-  CONSTRAINT `oh_session_audit_ibfk_1` FOREIGN KEY (`SEA_US_ID_A`) REFERENCES `oh_user` (`US_ID_A`)
+  KEY `FK_SESSION_AUDIT_USER_IDX` (`SEA_US_ID_A`),
```

Comment:
To match the sql 04 load make this change:
```suggestion
  KEY `FK_SESSION_AUDIT_USER` (`SEA_US_ID_A`),
```
If that is unacceptable then modify line 64 of step_95 to include the `_IDX` suffix.

Reply:
This issue is still in the current file after the corrections for syntax errors in the load demo data script.

Reply:
sorry, my bad, I thought it was a mysql feature but it is not. Find the way to make them identical